Kudu.Services/FetchHelpers/OneDriveHelper.cs

```diff
@@ -62,7 +62,7 @@ internal ILogger Logger
             ChangeSet changeSet = null;
             string cursor = _settings.GetValue(CursorKey);
             ChangesResult changes = null;
-            using (_tracer.Step("Getting delta changes with cursor: {0} ...", cursor))
+            using (_tracer.Step("Getting delta changes with cursor: {0}...", cursor.Truncate(5)))
```

Comment:
Here it is:
![image](https://cloud.githubusercontent.com/assets/6472374/15980022/a12abe8e-2f70-11e6-9cb8-06836ef23c90.png)

![image](https://cloud.githubusercontent.com/assets/6472374/15980037/b3f21846-2f70-11e6-84b3-b838a39497cf.png)